api/src/openapi/schemas/biohub-data-submission.ts

```diff
@@ -19,40 +19,27 @@ export const BioHubDataSubmission = {
     features: {
       type: 'array',
       items: {
-        $ref: '#/$defs/Feature'
```

Comment:
In general: If we want to use `$refs` in our openapi spec, the only caveat is that the `$defs` section has to be included in the `root-api-doc.ts` (because `$defs` is a top level section that can only exist once).